Assets/HoloToolkit/UX/Scripts/Lines/LineBaseEditor.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity;
+using UnityEngine;
+using MRTK.UX;
+#if UNITY_EDITOR
+using UnityEditor;
+
+/// <summary>
+/// Base class for line editors (splines, bezeirs, parabolas, etc)
+/// </summary>
+public class LineBaseEditor : MRTKEditor
```

Comment:
Editor scripts should live in an Editor folder

Reply:
This extends [MRTKEditor](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Utilities/Scripts/Inspectors/MRTKEditor.cs), which must be kept outside Editor folders in order to function.

Reply:
I really _really_ don't like this pattern.

Reply:
Yes I can tell it makes your skin crawl, lol. Maybe we should open a dedicated issue to discuss the pros / cons.

Reply:
> your skin crawl

🤣 I wouldn't go _that_ far, but yes. It goes against all the industry standard practices.

Reply:
>It goes against all the industry standard best practices.

Unity-land is a weird place, man. Sometimes down is up and black is white. I long for the clarity of a pure C# repo.

Reply:
Haha, Unity development is a different beast. I think a lot of traditional software engineers get tripped up with some things.